plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -265,16 +265,19 @@ export class UserInteractionInstrumentation extends InstrumentationBase<unknown>
       ) {
         const once = useCapture && useCapture.once;
         const patchedListener = (...args: any[]) => {
-          const target = this;
           let parentSpan: api.Span | undefined;
           const event: Event | undefined = args[0];
+          const target = event?.target;
           if (event) {
             parentSpan = plugin._eventsSpanMap.get(event);
           }
           if (once) {
             plugin.removePatchedListener(this, type, listener);
           }
-          const span = plugin._createSpan(target, type, parentSpan);
+          const span =
+            target instanceof HTMLElement
```

Comment:
`_createSpan` already has a number of "does this have enough information to produce a good span? -> if not, `undefined`" checks, and I think this one belongs inside `_createSpan` rather than outside it, to keep said logic centralized.